server/homebrew.api.js

```diff
@@ -281,35 +274,33 @@ const api = {
 
 		res.status(200).send(saved);
 	},
-	getBrewThemeWithCSS : async (req, res)=>{
+	//Return CSS for a brew theme, with @include endpoint for its parent theme if any
+	getBrewThemeCSS : async (req, res)=>{
 		const brew = req.brew;
 		splitTextStyleAndMetadata(brew);
 		res.setHeader('Content-Type', 'text/css');
-		const themePath = themes[_.upperFirst(req.brew.renderer)].hasOwnProperty(req.brew.theme) ? `/css/${req.brew.renderer}/${req.brew.theme}` : `/css/${req.brew.theme}`;
-		// Drop Parent theme if it has already been loaded.
-		// This assumes the continued use of the V3/5ePHB and V3/Blank themes for the app.
-		const parentThemeImport = ((req.brew.theme != '5ePHB') && (req.brew.theme != 'Blank')) ? `@import url(\"${themePath}\");\n\n`:'';
```

Comment:
This was in here because the editor loads this theme and double loading was causing issues.